app/ws/RadioTower.js

```diff
@@ -85,13 +85,14 @@ module.exports = {
       let roomName = ''
       if ( message.streamId && message.streamId.trim( ) !== '' ) {
         roomName = `stream-${message.streamId}`
-      } else if ( !( !message.resourceId || !message.resourceType ) )
+      } else if ( !( !message.resourceId || !message.resourceType ) ) {
         roomName = `${message.resourceType}-${message.resourceId}`
+      }
 
-      if ( roomName === '' )
+      if ( roomName === '' ) {
         return winston.debug( `Failed to deliver broadcast from ${senderClientId} (no room name).` )
-
-      winston.debug( `📣 broadcast in ${roomName} from ${senderClientId}: ${message.args.eventType}` )
```

Comment:
This is where I saw a bug. After reading your comments I'm still not sure whether args should have an event type as it's mean to be an `anything that can be jsonified` object. Regardless, seems a shame that a debug log would kill a process from running 😄

Reply:
Awwww yes 🙈